zetaclient/tss/tss_signer.go

```diff
@@ -558,6 +573,8 @@ func GetTssAddrEVM(tssPubkey string) (ethcommon.Address, error) {
 	return keyAddr, nil
 }
 
+// TestKeysign tests the keysign
```

Comment:
```suggestion
// TestKeysign tests the keysign 
//  It is called when a new TSS is generated to ensure the network works as expected.
```

Maybe we can also add some details on what it actually does ?